src/Archives/ArchiveUnpacker.h

```diff
@@ -19,23 +19,23 @@ namespace Archives
 
 		std::string GetVolumeFilename() { return m_ArchiveFilename; };
 		uint64_t GetVolumeFileSize() { return m_ArchiveFileSize; };
-		std::size_t GetNumberOfPackedFiles() { return m_NumberOfPackedItems; };
-		bool ContainsItem(const std::string& internalName);
-		std::size_t GetInternalItemIndex(const std::string& internalName);
-		void ExtractFile(const std::string& internalFilename, const std::string& pathOut);
+		std::size_t GetPackedItemCount() { return m_PackedItemCount; };
+		bool Contains(const std::string& name);
+		std::size_t GetIndex(const std::string& name);
+		void ExtractFile(const std::string& name, const std::string& pathOut);
 
-		virtual std::string GetInternalName(std::size_t index) = 0;
-		virtual uint32_t GetInternalItemSize(std::size_t index) = 0;
+		virtual std::string GetName(std::size_t index) = 0;
+		virtual uint32_t GetSize(std::size_t index) = 0;
 		virtual void ExtractFile(std::size_t index, const std::string& pathOut) = 0;
 		virtual void ExtractAllFiles(const std::string& destDirectory);
 		virtual std::unique_ptr<SeekableStreamReader> OpenStream(std::size_t index) = 0;
-		virtual std::unique_ptr<SeekableStreamReader> OpenStream(const std::string& internalName);
+		virtual std::unique_ptr<SeekableStreamReader> OpenStream(const std::string& name);
 
 	protected:
-		void CheckPackedIndexBounds(std::size_t index);
+		void CheckIndexBounds(std::size_t index);
```

Comment:
Yes, actually, I think I like removing "Packed" as well. I suppose "Packed" is roughly equivalent to "Internal", which we were removing.

The previous API allowed for different filenames inside the archive from the source file being packed, which was one reason to distinguish between internal names and external names, or packed files versus unpacked files. Now that the API has been simplified, this distinction no longer seems necessary. Or rather, it is handled by a couple of methods with `name` and `pathOut` to mark the distinction.